pkg/portForward/kubeportforward/portForward_test.go

```diff
@@ -23,6 +23,7 @@ func Test_getCompleteCustomPortPairs(t *testing.T) {
 			args: args{
 				definedPorts: []api.ForwardedPort{
 					{ContainerName: "runtime", LocalPort: 8080, ContainerPort: 8000},
+					{ContainerName: "tools", LocalPort: 5000, ContainerPort: 5000},
```

Comment:
Previously, there were 2 non-customized ports, and a random value was assigned to 5000 and 9000 from time to time, sometimes 20001 would be assigned to 5000 and 20002 to 9000 and vice versa.

Adding 5000 to the definedPorts will help avoid this flake while also maintaining the test integrity.